openfisca_us/variables/demographic/household.py

```diff
@@ -209,3 +209,20 @@ class county(Variable):
     entity = Household
     label = u"County"
     definition_period = ETERNITY
+
+
+class Provider_Type(Enum):
+    DCC_SACC = "Licenced/registered/permitted day care center; registered school-age child care"
+    FDC_GFDC = "Registered family day care homes; licensed group family day care"
+    LE_GC = "Legally exempt group child care programs"
+    LE_STD = "Informal child care standard rate"
+    LE_ENH = "Informal child care enhanced rate"
+
+
+class provider_type(Variable):
+    value_type = Enum
+    possible_values = Provider_Type
+    default_value = Provider_Type.DCC_SACC
```

Comment:
Could you also add a comment justifying the default? If it's just the first one listed that's fine, but all else equal if we can back up that it's the most common that's preferable.
```suggestion
    default_value = ProviderType.DCC_SACC
```

Reply:
I actually just assumed that would be the most common one. Do we need to have default value?

Reply:
Actually based on this table, DCC and SACC together is the most common one (in NY at least).
<img width="553" alt="Screen Shot 2021-10-29 at 8 19 09 PM" src="https://user-images.githubusercontent.com/26745287/139515249-62db1e1f-46a7-4e5a-9dee-c5a0048357d1.png">
https://ocfs.ny.gov/main/reports/2019-Child-Care-Market-Rate-Survey.pdf (page 21)